src/REPLMode/command_declarations.jl

```diff
@@ -153,7 +153,7 @@ precompiled before, or the precompile cache has been deleted by the LRU cache st
 (see `JULIA_MAX_NUM_PRECOMPILE_FILES`).
 
 **Examples**
-```
+```julia-repl
```

Comment:
Shouldn't this be text or some other neutral "language"? This isn't  really julia-repl.

Reply:
Ah I expected it to detect the other repl modes and color appropriately.

Reply:
Hmm, we could extend

https://github.com/JuliaLang/julia/blob/19165bec4a3e2ed0de4f2f43bccc3fcac2d76662/stdlib/Markdown/src/render/terminal/render.jl#L123-L124

or it might be nice to make it possible for other "languages" to register their own highlighting functions (with a Markdown + REPL pkg ext for instance) then you could have `pkg-repl` for instance.